### PR TITLE
📝Source JIra: updated info in spec and docs

### PR DESCRIPTION
**Pull request by darynaishchenko**
Added more specific info for API token, email, start_date and projects fields in spec and docs.

### CODE REVIEW

File: docs/integrations/sources/jira.md [hunk 1]
```diff
@@ -124,35 +124,36 @@ The Jira connector should not run into Jira API limitations under normal usage.
...
-| 0.2.4   |            |                                                            | Implementing base\_read acceptance test dived by stream groups.                                                         |
-| 0.2.3   |            |                                                            | Implementing incremental sync. Migrated to airbyte-cdk. Adding all available entities in Jira Cloud.                    |
+| Version | Date        | Pull Request                                               | Subject                                                                                                                 |
+|:--------|:------------|:-----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
+| 0.3.9   | 2023-05-016 | [\#26114](https://github.com/airbytehq/airbyte/pull/26114) | Update fields info in docs and spec                                                                                     |
```
Comment (lazebnyi) [inline docs/integrations/sources/jira.md:129]:
```suggestion
| 0.3.9   | 2023-05-16 | [\#26114](https://github.com/airbytehq/airbyte/pull/26114) | Update fields info in docs and spec                                                                                     |
```

**Review by lazebnyi — APPROVED** [review]
LGTM!

File: docs/integrations/sources/jira.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ The Jira connector should not run into Jira API limitations under normal usage.
 
 | Version | Date        | Pull Request                                               | Subject                                                                                                                 |
 |:--------|:------------|:-----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
-| 0.3.9   | 2023-05-016 | [\#26114](https://github.com/airbytehq/airbyte/pull/26114) | Update fields info in docs and spec                                                                                     |
+| 0.3.9   | 2023-05-16 | [\#26114](https://github.com/airbytehq/airbyte/pull/26114) | Update fields info in docs and spec                                                                                     |
```
Comment (lazebnyi) [inline docs/integrations/sources/jira.md:129]: Need add extra space after `2023-05-16`.

**Review by sherifnada — APPROVED** [review]

File: airbyte-integrations/connectors/source-jira/source_jira/spec.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
       "email": {
         "type": "string",
         "title": "Email",
-        "description": "The user email for your Jira account.",
+        "description": "The user email for your Jira account. Email is used for Authorization to your account by BasicAuth.",
```
[the remark this answers is not in the document]
Comment (darynaishchenko) [reply airbyte-integrations/connectors/source-jira/source_jira/spec.json]: thanks for suggestion, we checked it with unblockers, it's make sense

**Pull request merged**